tests/fixtures/mutant/deliverables.yaml

```
@@ -1,2191 +1,2198 @@
 files:
 - format: txt
   id: maturejay
```

```suggestion
  id: case_id
```

Generalise the case/analysis IDs